.github/workflows/python.yml

```diff
@@ -73,7 +73,7 @@ jobs:
       matrix:
         python: [2.7, 3.6]
     env:
-      PYTHON_VERSION: ${{ matrix.python }}
+      PYTHON: ${{ matrix.python }}
```

Comment:
I first thought to do this as well, but in all other places (like cmake config etc), we are using PYTHON_VERSION and not PYTHON, so therefore opted to change it here

Reply:
The docker build time parameters are the docker `ARG`s. We want to install the appropriate python version from Conda in build time, so we need to pass the python version as an `ARG`. 

The `python` arg is defined in the [docker-compose file](https://github.com/apache/arrow/blob/master/docker-compose.yml#L289) and it refers to the `$PYTHON` environment variable, which is read by docker-compose. The default values for the docker-compose environment variables are defined in the .env file.